codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2Headers.java

```diff
@@ -146,6 +146,12 @@ public CharSequence status() {
     }
 
     @Override
+    public Http2Headers clear() {
```

Comment:
Happy to do that though I will have to revert the benchmarks to stop using 'clear' and instead create a new object each time to copy into.